modules/ossm-config-sampling.adoc

```diff
@@ -5,18 +5,23 @@
 [id="ossm-config-sampling_{context}"]
 = Adjusting the sampling rate
 
-The distributed tracing sampling rate is set to sample 100% of traces in your service mesh by default. A high sampling rate consumes cluster resources and performance but is useful when debugging issues. Before you deploy {ProductName} in production, set the value to a smaller proportion of traces.
+A trace is an execution path between services in the service mesh. A trace is comprised of one or more spans. A span is a logical unit of work that has a name, start time, and duration. The sampling rate determines how often a trace is generated.
 
-A trace is an execution path between services in the service mesh. A trace is comprised of one or more spans. A span is a logical unit of work that has a name, start time, and duration.
+The Envoy proxy sampling rate is set to sample 100% of traces in your service mesh by default. A high sampling rate consumes cluster resources and performance but is useful when debugging issues. Before you deploy {ProductName} in production, set the value to a smaller proportion of traces. For example, set `spec.tracing.sampling` to `100.`
 
-The sampling rate determines how often a trace is generated. Configure sampling as a scaled integer representing 0.01% increments.
+Configure the Envoy proxy sampling rate as a scaled integer representing 0.01% increments.
 
 In a basic installation, `spec.tracing.sampling` is set to `10000`, which samples 100% of traces. For example:
 
 * Setting the value to 10 samples 0.1% of traces.
 * Setting the value to 500 samples 5% of traces.
 
-Setting the value to `10000` is useful for debugging, but can affect performance. For production, set `spec.tracing.sampling` to `100.`
+[NOTE]
+====
+The Envoy proxy sampling rate applies for applications that are available to a Service Mesh, and use the Envoy proxy. This sampling rate determines how much data the Envoy proxy collects and tracks.
+
+The Jaeger remote sampling rate applies to applications that are external to the Service Mesh, and do not use the Envoy proxy, such as a database. This sampling rate determines how much data the Jaeger agent collects and tracks. For more information, see xref:../../jaeger/jaeger_install/rhbjaeger-deploying.adoc#jaeger-config-sampling_jaeger-deploying[Configurating Jaeger sampling].
```

Comment:
> This sampling rate determines how much data the Jaeger agent collects and tracks

This sampling rate determines how much data the Jaeger system collects and stores.

Reply:
"This sampling rate determines how much data the distributed tracing system collects and stores."  (So that I don't have to update this later.  =D  )

Reply:
Thanks for catching this, @JStickler. I made this change.